src/Artemis.UI/Screens/Device/Tabs/DeviceLayoutTabView.axaml.cs

```diff
@@ -16,13 +16,13 @@ public DeviceLayoutTabView()
 
     private void LayoutPathButton_OnClick(object? sender, RoutedEventArgs e)
     {
-        TopLevel.GetTopLevel(this).Clipboard.SetTextAsync(ViewModel.DefaultLayoutPath);
+        TopLevel.GetTopLevel(this).Clipboard.SetTextAsync(ViewModel?.DefaultLayoutPath ?? string.Empty);
```

Comment:
Better to early return if it is null than to set an empty string to the cliptboard